src/apps/answers/service.py

```diff
@@ -366,6 +366,18 @@ async def validate_multiinformant_assessment(
             activity, flow = await asyncio.gather(activity_future, flow_future)
             if not activity and not flow:
                 raise MultiinformantAssessmentInvalidActivityOrFlow()
+            
+        is_admin = await AppletAccessCRUD(self.session).has_any_roles_for_applet(
+            applet_id,
+            respondent_subject.user_id,
+            [Role.OWNER],
+        )
+
+        if not is_admin:
+            if not target_subject or not source_subject:
+                raise MultiinformantAssessmentNoAccessApplet()
+            
+            await self._validate_relation_between_subjects_in_applet(respondent_subject, target_subject, source_subject)
```

Comment:
Admins that can perform this will have the roll of `Role.OWNER` and `Role.MANAGER`. You should be able to call `CheckAccessService.check_applet_manager_list_access` like it was previously doing above.  Something like this

```suggestion
try:
    await CheckAccessService(self.session, self.user_id).check_applet_manager_list_access(applet_id)
    is_admin = True
except AppletAccessDenied:
    is_admin = False

if not is_admin:
    if not target_subject or not source_subject:
        raise MultiinformantAssessmentNoAccessApplet()

    await self._validate_relation_between_subjects_in_applet(respondent_subject, target_subject, source_subject)
```

Reply:
yes the function you mention seems to be doing the right work!
thank you for the letting me know about it !